issue-scan

```diff
@@ -134,42 +138,50 @@ def output_task(command, issue, repo, verbose):
     if "pull_request" in issue or "commits" in issue:
         ref = f"pull/{number}/head"
         sha = api.get(f"git/ref/{ref}")["object"]["sha"]
+        pull = number
     else:
         ref = testmap.get_default_branch(repo)
         sha = api.get(f"git/ref/heads/{ref}")["object"]["sha"]
-
-    if verbose:
-        return f"issue-{number} {name} {context} {ref}"
+        pull = None
 
     current = time.strftime('%Y%m%d-%H%M%S')
+    slug = f'{name}-{context}-{sha[:8]}-{current}'
+
     cmd = f"./s3-streamer --repo {repo} --test-name {name}-{context}-{current} " \
         f"--github-context {name}/{context} --revision {sha} -- sh -exc '"
     cmd += f"./make-checkout --verbose --repo {repo} {ref}; cd make-checkout-workdir; "
     cmd += f"./{name} --verbose --issue={number} {context}'"
 
-    return cmd
-
-
-def queue_task(channel, result):
-    body = {
-        "command": result,
+    return {
+        'job': {
+            'repo': repo,
+            'sha': sha,
+            'pull': pull,
+            'slug': slug,
+            'context': f'{name}/{context}',
+            'command': (f'./{name}', '--verbose', f'--issue={number}', context),
+            'secrets': ('github-token', 'image-upload'),
+        },
+        'human': f"issue-{number} {name} {context} {ref}",
+        "command": cmd,
         "type": "issue",
     }
-    queue = 'rhel' if contains_any(result, REDHAT_TASKS) else 'public'
+
+
+def queue_task(channel, body):
+    queue = 'rhel' if contains_any(body['job']['context'], REDHAT_TASKS) else 'public'
     channel.basic_publish('', queue, json.dumps(body),
                           properties=pika.BasicProperties(priority=distributed_queue.MAX_PRIORITY))
-    logging.info("Published issue task with command: '%s'", result)
+    logging.info("Published issue task with command: '%s'", body['command'])
```

Comment:
```suggestion
 logging.info("Published issue task: '%s'", body)
```
so that we see the JSON too. Grabbing them from the log and stuffing them into `job-runner` feels useful

Reply:
Already did that, but with `%r` and without quotes.